local/owr_media_renderer.c

```diff
@@ -85,7 +85,6 @@ static void owr_media_renderer_finalize(GObject *object)
 
     if (priv->pipeline) {
         gst_element_set_state(priv->pipeline, GST_STATE_NULL);
-        gst_bin_remove(GST_BIN(priv->pipeline), priv->sink);
```

Comment:
This shouldn't make a difference unless priv->sink==NULL. But is correct. If needed it will remove the sink in unref below